### PR TITLE
Obsdef 13989 wizard wrapper

### PR DESCRIPTION
**Pull request by Malayvarma27**
Testing link : http://10.249.253.4:3000
key: eyJhbGciOiJSUzI1NiIsImtpZCI6IlloNmpYd3RRcUMwQTlVUmRac005ZlFLS0k3VlI5S2Jvc3BrRUdScVRjQ0EifQ.eyJpc3MiOiJrdWJlcm5ldGVzL3NlcnZpY2VhY2NvdW50Iiwia3ViZXJuZXRlcy5pby9zZXJ2aWNlYWNjb3VudC9uYW1lc3BhY2UiOiJzdmMtb2JqZWN0c2NhbGUtZG9tYWluLWMyMCIsImt1YmVybmV0ZXMuaW8vc2VydmljZWFjY291bnQvc2VjcmV0Lm5hbWUiOiJvYmplY3RzY2FsZS1vcGVyYXRvci10b2tlbi1wa2h4MiIsImt1YmVybmV0ZXMuaW8vc2VydmljZWFjY291bnQvc2VydmljZS1hY2NvdW50Lm5hbWUiOiJvYmplY3RzY2FsZS1vcGVyYXRvciIsImt1YmVybmV0ZXMuaW8vc2VydmljZWFjY291bnQvc2VydmljZS1hY2NvdW50LnVpZCI6IjZiYmNmNDk3LTgzOWYtNDZjMC1hMzQ5LTg3ZTkzY2UwN2Y5NCIsInN1YiI6InN5c3RlbTpzZXJ2aWNlYWNjb3VudDpzdmMtb2JqZWN0c2NhbGUtZG9tYWluLWMyMDpvYmplY3RzY2FsZS1vcGVyYXRvciJ9.MzDyDe3E8WjsW8dyz9Y2bNEKA3VgS7DPi4ScjUgr0FMHGuLekUMHS_4PGDJV52nuF1YOzD-lUrlQ7n-BXsY532Gd1WVK9cg1ykfj7Zlfqqm4Sp9DVUIH9CDB3E9B1V-iMstEgytmH1uvoOuBn1RPzLUmHacOu8VjkG05IEZPtYyEyPIu8jSi5euEqJ-rfBE-kTPAwCExYVkmvmGWg5f2dBdqLInY60jjXLpliFocELkXft_GhXzu3mGbMSaWQ2qhOwYtn-SjppVJNzfgQRmMyDPhPvIFaVNIZy9Mn_MAVd7XWkfP7r4oBl3VsaptB8vWSx7W5_g-NulvZU8vxAMEAg

WizardWrapper

### CODE REVIEW

**Comment by tejasjc-ecs** [comment]
Tested using given link.
Couldn't find this border on Active step in DDS, can we recheck this? also it seems to be overlapping with vertical border.
 
image

File: src/components/wizard/Wizard.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,514 @@
...
+     if (action === "insert") {
+         // Insert new step
+         steps.splice(index, 0, {index: index, name: "page new", type: WizardStepType.SUB_STEP});
+     } else if (action === "remove") {
+         steps.splice(index, 1);
+     }
+     return buildStepsUI();
+ }
+ 
+ const store = new Store({
```
[the remark this answers is not in the document]
Comment (Malayvarma27) [reply src/components/wizard/Wizard.stories.tsx]: removed the focus outline

File: src/styles/components/Wizard.scss [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+            margin-top: 0;
+
+            .clr-input-wrapper {
+                .clr-subtext {
+                    text-align: left;
+                }
+            }
+        }
+    }
+}
```
[the remark this answers is not in the document]
Comment (Malayvarma27) [reply src/styles/components/Wizard.scss]: done

**Review by akate1 — APPROVED** [review]

**Review by tejasjc-ecs — APPROVED** [review]
Tested using given link.
LGTM

**Comment by GauravMenezes-Professional** [comment]
I noticed this happening in Wizard (ObjectStore Create):

image

However, this does not seem to happen in a normal text box (Checked Bucket Tab in ObjectStore Details):

image

**Pull request merged**